packages/plugin-defender/src/hardhat/utils.ts

```diff
@@ -0,0 +1,7 @@
+import { ProposalResponse } from 'defender-admin-client';
+
+const BaseUrl = 'https://defender.openzeppelin.com';
+
+export function getProposalUrl(proposal: ProposalResponse): string {
+  return `${BaseUrl}/#/admin/contracts/${proposal.contractId}/proposals/${proposal.proposalId}`;
```

Comment:
I don't remember if I mentioned this already. I would've expected the server to send this URL as part of the response, rather than us building it based on the parameters.

If the URL came in the response, I'd imagine this gives Defender more flexibility to change things independently of this package.

Another alternative would be to build the URL in the client library, although if I remember correctly the client is quite thin.

What do you think?

Reply:
Moved to client in https://github.com/OpenZeppelin/defender-client/pull/29. Will remove from here once merged.